docs/Automation.md

```diff
@@ -1,183 +1,198 @@
+##### `CreateScanner`
+Create an object that implements `IScanner` using an instance of `Config`.
+
+###### Parameters
+
+The `ScannerFactory.CreateScanner` method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
-ScanResultsPassed | `int` | The count of elements that passed all accessibility scans.
-ScanResultsFailed | `int` | The count of elements that failed one or more accessibility scans.
-ScanResultsInconclusive | `int` | The count of elements that registered an inconclusive scan—these are typically cases where a human needs to evaluate the control and determine whether or not a fix is necessary.
-ScanResultsUnsupported  | `int` | The count of elements that Accessibility Insights is unable to scan—these are typically elements like embedded web controls.
-ScanResultsTotal | `int` | The sum of ScanResultsPassed, ScanResultsFailed, ScanResultsInconclusive, and ScanResultsUnsupported.
+config | `Config` | The configuration used by the returned `IScanner` object.
 
-#### Stop Command
-The Stop command terminates an automation session. State and associated
-resources are freed. The test framework can then exit or create another session
-(via the Start command).
+###### Return object
 
-##### Inputs
+The `ScannerFactory.CreateScanner` method returns an `IScanner` object.
 
-###### .NET
+#### IScanner
 
-The **StopCommand.Execute** method accepts no parameters.
+##### `Scan`
+The Scan runs AxeWindows automated tests using the config provided at the time of creation of the scanner.
 
-###### PowerShell
+###### Parameters
 
-The **Stop-AxeWindows** cmdlet accepts no parameters
+The `Scan` method accepts no parameters.
 
-##### Return object
-The StopCommandResult object has the following properties:
+###### Return object
+
+The `Scan` method returns a `ScanResults` object and has the following properties:
 
 **Name** | **Type** | **Description**
 ---|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
+ErrorCount | `int` | A count of all errors across all elements scanned.
+Errors | `IEnumerable<ScanResult>` | A collection of errors found during the scan.
+OutputFile | `(string A11yTest, string Sarif)` | A Tuple with paths to any output files written as a result of a scan. Tuple members are A11yTest and Sarif (**not implemented yet**).
```

Comment:
>Tuple [](start = 51, length = 5)

We should discuss this. The Tuple was changed in [https://github.com/microsoft/axe-windows/pull/91](https://github.com/microsoft/axe-windows/pull/91). My recommendation is to document OutputFile as a struct with fields called A11yTest and Sarif. The Tuple then becomes an implementation detail independent of the consuming code.